sbnobj/Common/Calibration/classes_def.xml

```diff
@@ -5,9 +5,15 @@
   <class name="sbn::TrackTruth" ClassVersion="10">
    <version ClassVersion="10" checksum="2749399767"/>
   </class>
-  <class name="sbn::TrueParticle" ClassVersion="10">
+  <class name="sbn::TrueParticle" ClassVersion="12">
+   <version ClassVersion="12" checksum="855399631"/>
+   <version ClassVersion="11" checksum="3478510212"/>
```

Comment:
Please remove transient versions (if you implement some of the changes I have recommended, there will be more).
```suggestion
  <class name="sbn::TrueParticle" ClassVersion="11">
   <version ClassVersion="11" checksum="855399631"/>
```